sdk/Lusid.Sdk.Tests/tutorials/Instruments/InterestRateSwapExamples.cs

```diff
@@ -123,5 +123,124 @@ public void InterestRateSwapPortfolioCashFlowsExample(ModelSelection.ModelEnum m
             var irs = InstrumentExamples.CreateExampleInterestRateSwap("Vanilla");
             CallLusidGetPortfolioCashFlowsEndpoint(irs, model);
         }
+
+        [TestCase(ModelSelection.ModelEnum.ConstantTimeValueOfMoney)]
```

Comment:
think you'll need to add a lusid feature number

Reply:
added